tsdb/wal/wal.go

```diff
@@ -782,7 +782,7 @@ func listSegments(dir string) (refs []segmentRef, err error) {
 	})
 	for i := 0; i < len(refs)-1; i++ {
 		if refs[i].index+1 != refs[i+1].index {
-			return nil, errors.New("segments are not sequential")
+			return nil, errors.Errorf("list segments in dir %s: %s", dir, errors.New("segments are not sequential"))
```

Comment:
```suggestion
			return nil, errors.Errorf("list segments in dir %s: %s", dir, "segments are not sequential")
```

Reply:
Poking @johncming in case you missed this. Let's get this merged!

Reply:
ping (: